lib/interfaces/swagger-custom-options.interface.ts

```diff
@@ -1,3 +1,5 @@
+import { FastifySwaggerUiConfigOptions } from 'fastify-swagger';
```

Comment:
`fastify-swagger` is an **optional** package so we can't reference its types. We'd have to copy them over to this lib, similarly to what we did with express types (see interface below)